### PR TITLE
Upgrade to Cairo 1.1.0

### PR DESCRIPTION
**Pull request by tensojka**
Also gets rid of dependency on alexandria because they aren't yet ready for C1.1

### CODE REVIEW

File: src/proposals.cairo [hunk 1]
```diff
@@ -204,9 +116,9 @@ mod Proposals {
         if index >= calldata_span.len() {
             return hashed_data;
         } else {
-            //let calldata_span: Span<(ContractAddress, u128)> = calldata.span();
             let (a, b) = *calldata_span.at(index);
-            let hashed_data = LegacyHash::hash(contract_address_to_felt252(a), b.into());
+            let b_felt: felt252 = b.into();
```
Comment (StarkFishinator) [inline src/proposals.cairo:120]: why this change?

**Review by StarkFishinator — COMMENTED** [review]
Most of the changes seems to be refactoring of types and formatting. We think it does not break anything, and it does not interfere with our governance code.

File: src/proposals.cairo (hunk 1 above)
Comment (tensojka) [reply src/proposals.cairo]: Good catch. During the upgrade this wasn't compiling for some reason (unable to infer type annotations in parts of code not yet upgraded that also didn't compile) and this fixed it.

Done
